ecosystem/python/sdk/aptos_sdk/async_client.py

```diff
@@ -527,6 +784,72 @@ async def transfer_object(
         )
         return await self.submit_bcs_transaction(signed_transaction)
 
+    async def view(
+        self,
+        function: str,
+        type_arguments: List[str],
+        arguments: List[str],
+        ledger_version: Optional[int] = None,
+    ) -> bytes:
+        """
+        Execute a view Move function with the given parameters and return its execution result.
+
+        The Aptos nodes prune account state history, via a configurable time window. If the requested ledger version
+        has been pruned, the server responds with a 410.
+
+        :param function: Entry function id is string representation of an entry function defined on-chain.
+        :param type_arguments: Type arguments of the function.
+        :param arguments: Arguments of the function.
+        :param ledger_version: Ledger version to get state of account. If not provided, it will be the latest version.
+        :returns: Execution result.
+        """
+        response = await self._post(
+            endpoint="view",
+            params={
+                "ledger_version": ledger_version,
+            },
+            headers={
+                "Accept": "application/json, application/x-bcs",
```

Comment:
I would probably just leave this to `applicaiton/json` for now.  Mainnet doesn't have BCS support quite yet, and I'm sure it will probably change abruptly at some point (when both are available, it'll just switch to bcs on you.